harness/determined/experimental/_native.py

```diff
@@ -192,13 +192,16 @@ def test_one_batch(
     trial_class: Optional[Type[det.Trial]] = None,
     config: Optional[Dict[str, Any]] = None,
 ) -> Any:
+
     # Override the scheduling_unit value to 1.
     # TODO(DET-2931): Make the validation step a single batch as well.
     config = {**(config or {}), "scheduling_unit": 1}
 
     logging.info("Running a minimal test experiment locally")
     checkpoint_dir = tempfile.TemporaryDirectory()
-    env, rendezvous_info, hvd_config = det._make_local_execution_env(True, config, limit_gpus=1)
+    env, rendezvous_info, hvd_config = det._make_local_execution_env(
+        True, config, limit_gpus=1, test_mode=True
```

Comment:
while you are here, can you switch this to be all keyword arguments?  That initial bare `True` is a little opaque.